pipelines/internal/commands/watch/watch.go

```diff
@@ -32,16 +36,20 @@ var (
 
 	actions = flags.Bool("actions", false, "show action details")
 	details = flags.Bool("details", false, "show event details")
+	topic   = flags.String("topic", "", "the Pub/Sub topic to watch")
 )
 
 func Invoke(ctx context.Context, service *genomics.Service, project string, arguments []string) error {
 	names := common.ParseFlags(flags, arguments)
 	if len(names) < 1 {
 		return errors.New("missing operation name")
 	}
+	if *topic == "" {
```

Comment:
We will still want to support polling and I think this doesn't need to be passed: when the operation data is fetched initially if there is a topic we'll start listening instead of polling.

Reply:
done